### DISCUSSION TITLE
Editor break time suggestions

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#28645] [Ideas] [answered]
While i like that now break time is added automatically in lazer, there are some things that i think should be added to this part of the editor.

1. **Make break time manually removable** - while maybe somewhere it is possible to edit, couldn't find such a option, and its important to make such features easily available. Since break time is added automatically, it can be added in places you dont want it too be, and since you can't remove it, you'll have to use stable to delete it, or remake the part of the map so it will delete itself.

2. **Make break time manually addable** - I doubt it will be needed THAT much, but when the automatic add doesn't work for whatever reason, its good to have a failsafe. A way to do this, is just make break time, whatever manually or automatically added, display it the Timing menu, so it can also be editable there, added and deleted. Basically making it work the same as Kiai would be the best option in my opinion

2. **'Break ends * late'** - This warning/problem message in the Verify menu isn't bad, but the way it links to the timeline is. When ecouring the same problem, exept when the breaks starts early, it makes sense to be sent to the start of it when you click at it, the same as any other problem, but when the break ends late, it also sends you to the start of the break??? this isnt THAT big of a 'problem', since you can just scroll there at this point, but it would make more sense to be sent to the end of it, aka where the problem actually is.[/list]

There isn't much else to list, because there is literally almost nothing to add, since its just a break. In my opinion these changes would be usefull when editing breaks and would make the lazer editor even better, not much, but still better.

**Comment by peppy** [comment, marked as answer]
dunno about manual add of break time. it's already adding wherever it can.

as for removal, i guess we can add it?

**Reply by Tenexxt** [reply]
Yea, this post is mostly my nitpicking, literally small details. Thanks for the reply tho
